package.json

```diff
@@ -55,6 +55,7 @@
 	},
 	"devDependencies": {
 		"eslint": "^7.24.0",
+		"gtfs-via-postgres": "^4.1.0",
```

Comment:
Is there a reason for this change?

Reply:
This was added accidentally. Removed it.